notify/opsgenie/opsgenie.go

```diff
@@ -211,13 +219,44 @@ func (n *Notifier) createRequests(ctx context.Context, as ...*types.Alert) ([]*h
 			responders = append(responders, responder)
 		}
 
+		var visibleTos []opsGenieCreateMessageVisibleTo
+		for _, v := range n.conf.VisibleTo {
+			visibleTo := opsGenieCreateMessageVisibleTo{
+				ID:       tmpl(v.ID),
+				Name:     tmpl(v.Name),
+				Username: tmpl(v.Username),
+				Type:     tmpl(v.Type),
+			}
+
+			if visibleTo == (opsGenieCreateMessageVisibleTo{}) {
+				// Filter out empty responders. This is useful if you want to fill
+				// responders dynamically from alert's common labels.
+				continue
+			}
+
+			if visibleTo.Type == "teams" {
+				teams := safeSplit(visibleTo.Name, ",")
+				for _, team := range teams {
+					newVisibleTo := opsGenieCreateMessageVisibleTo{
+						Name: tmpl(team),
+						Type: tmpl("team"),
```

Comment:
This doesn't make sense to me. You could template it back to `user` after we've checked that it is `teams`.

Reply:
I think it should just be `Type: "team"`. Fixed in new commit.

Not sure if I understand the reference to `user` here?